docs/core/porting/index.md

```diff
@@ -13,35 +13,91 @@ You might have code that currently runs on the .NET Framework that you're intere
 
-4. Convert all of your `packages.config` dependencies to the [PackageReference](/nuget/consume-packages/package-references-in-project-files) format with the [conversion tool in Visual Studio](/nuget/consume-packages/migrate-packages-config-to-package-reference).
+1. If you are using Windows-specific APIs (such as registry access), you can install the [Windows Compatibility Pack](./windows-compat-pack.md).
+
+1. At this point, you can switch to targeting .NET Core (generally for applications) or .NET Standard (for libraries).
 
-   This step involves converting your dependencies from the legacy `packages.config` format. `packages.config` doesn't work on .NET Core, so this conversion is required if you have package dependencies.
+   If there are no preprocessor directives to conditional compile code for .NET Framework or .NET Standard, this will be as simple as finding the following in the project file:
 
-5. Create new projects for .NET Core and copy over source files, or attempt to convert your existing project file with a tool.
+   ```xml
+   <TargetFramework>net472</TargetFramework>
+   ```
 
-   .NET Core uses a simplified (and different) [project file format](../tools/csproj.md) than .NET Framework. You'll need to convert your project files into this format to continue.
+   and switch it to the desired framework. For .NET Core 3.1, this would be:
 
-6. Port your test code.
+   ```xml
+   <TargetFramework>netcoreapp3.1</TargetFramework>
+   ```
 
-   Because porting to .NET Core is such a significant change to your codebase, it's highly recommended to port your test projects so that you can run tests as you port your code over. MSTest, xUnit, and NUnit all work on .NET Core.
+   However, if this is a library that you want to continue supporting .NET Framework specific builds for some reason, you can [multi-target](../..//library-guidance/cross-platform-targeting.md) by replacing it with the following:
 
-Additionally, you can attempt to port smaller solutions or individual projects in one operation to the .NET Core project file format with the [dotnet try-convert](https://github.com/dotnet/try-convert) tool. `dotnet try-convert` is not guaranteed to work for all your projects, and it may cause subtle changes in behavior that you depended on. Use it as a _starting point_ that automates the basic things that can be automated. It isn't a guaranteed solution to migrating a project.
+   ```xml
+   <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
+   ```
 
 ## Next steps
 
 >[!div class="nextstepaction"]
 >[Analyze dependencies](third-party-deps.md)
+>[Package NuGet Package](../deploying/creating-nuget-packages.md)
```

Comment:
I realize that ASP.NET -> ASP.NET Core is only one specific type of migration, but I think it might be valuable to have links to our ASP.NET Core migration docs here because that transition is both common enough and more-complicated-than-other-migrations enough that a lot of people seeing this doc should probably also see those ones.

Reply:
Added